### PR TITLE
Jn tabbed interface

### PR DESCRIPTION
**Pull request by jniewiel**

### CODE REVIEW

File: config/routes.rb
Comment (aizatibraimova) [inline config/routes.rb:1]: Routes are clean and easy to follow.

File: app/controllers/users_controller.rb
Comment (aizatibraimova) [inline app/controllers/users_controller.rb:1]: Ordering the methods is really helpful to debug!

**Review by aizatibraimova — APPROVED** [review]

File: app/views/users/followers.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link" %>
+      </li>
+    </ul>
+  </div>
+</div>
+
+<% @user.followers.each do |follower| %>
+  <%= render "users/follow", follow: follower %>
```
Comment (aborov) [inline app/views/users/followers.html.erb:28]: It's smart to wrap your views for `followers` and `following` routes in a partial as they are basically the same

File: app/views/users/following.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <%= link_to "Liked photos", liked_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Feed", feed_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Followers", followers_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link" %>
```
Comment (aborov) [inline app/views/users/following.html.erb:21]: Great use of `link_to` helper method to keep your links reusable and consistent!

File: config/routes.rb [hunk 3]
```diff
@@ -8,6 +8,13 @@
   resources :follow_requests
   resources :likes
   resources :photos
+
+  get ":username/liked" => "users#liked", as: :liked
+  get ":username/feed" => "users#feed", as: :feed
+  get ":username/followers" => "users#followers", as: :followers
+  get ":username/following" => "users#following", as: :following
```
Comment (aborov) [inline config/routes.rb:15]: Definition of user-specific routes perfectly done!

**Review by aborov — APPROVED** [review]
Great work on your branch updates! Pleasure to read!

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, Jan!

In your "users" view folder you could also create a partial for all the tab info and use the partial in the feed, followers, following, liked and show view files. 

You can now merge these changes back to main.

**Pull request merged**